core/controlplane/config/templates/cloud-config-controller

```diff
@@ -198,7 +198,8 @@ coreos:
         --rkt-path=/usr/bin/rkt \
         --rkt-stage1-image=coreos.com/rkt/stage1-coreos \
         --node-labels node-role.kubernetes.io/master{{if .Experimental.NodeLabels.Enabled}},{{.Experimental.NodeLabels.String}} \
-        {{end}}--register-with-taints=node.alpha.kubernetes.io/role=master:NoSchedule \
```

Comment:
Ah, good catch!
So we had been encountering errors when no custom node labels are provided from an user by this:

```
--node-labels node-role.kubernetes.io/master{{if .Experimental.NodeLabels.Enabled}},{{.Experimental.NodeLabels.String}} \
{{end}}--register-with-taints=node.alpha.kubernetes.io/role=master:NoSchedule
```

rendered as:

```
--node-labels node-role.kubernetes.io/master--register-with-taints=node.alpha.kubernetes.io/role=master:NoSchedule
```